src/routes/(settings)/settings/+layout.svelte

```
@@ -1,47 +1,49 @@
 <script lang="ts">
   import { page } from "$app/state";
   import { Button } from "$lib/components/ui/button";
-  import { Sheet, SheetContent, SheetTrigger } from "$lib/components/ui/sheet";
   import {
+    ArchiveIcon,
     ArrowLeftIcon,
+    BotIcon,
     CreditCardIcon,
+    ExternalLinkIcon,
     HomeIcon,
```

ESLint

[error] 'HomeIcon' is defined but never used.

(@typescript-eslint/no-unused-vars)